packages/ui/src/app/pages/WorkingGroups/WorkingGroupsOpening.tsx

```diff
@@ -149,7 +149,10 @@ export const WorkingGroupOpening = () => {
               <StatusBadge />
             </BadgesRow>
             <Statistics>
-              <DurationStatistics title="Time Left" value={opening.expectedEnding} />
+              {
+                OpeningStatuses.CANCELLED ? "" :
```

Comment:
Do you try  only show when canceled?
Could be done in `DurationStatistics` instead with something like `if (!duraiton) return <></>`